databrowser/src/domain/datasets/ui/detailView/toolBox/DetailViewToolBox.vue

```diff
@@ -0,0 +1,65 @@
+<!--
+SPDX-FileCopyrightText: NOI Techpark <digital@noi.bz.it>
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+  <ToolBox>
+    <ToolBoxSection
+      :sectionKey="ToolBoxSectionKey.EXPORTS"
+      :title="t('datasets.toolBox.exportDatasets.panelName')"
+      :iconComponent="IconDownload"
+      info="Scarica i dati in diversi formati"
```

Comment:
Prop `info` is not used neither in `ToolboxSection.vue` nor in `ToolBox.vue`, please remove this prop